api/src/middleware/VerifyPhoneNumber.ts

```diff
@@ -0,0 +1,19 @@
+import PhoneNumber from "awesome-phonenumber";
+
+export const VerfyPhonNumber = ({number}): String => {
```

Comment:
Utilicemos sustantivos (nouns) en vez de verbos para nombrar objetos:

```suggestion
export const PhoneNumberVerificationMiddleware = ({number}): String => {
```

Reply:
Hagamos typing de los parámetros:

```suggestion
export const VerfyPhonNumber = ({number}: VerifyPhoneNumberCodeInput): String => {
```

Estos ya están definidos en `import { VerifyPhoneNumberCodeInput } from "@corecodeio/libraries/api";`